### PR TITLE
Several small documentation fixes

### PR DESCRIPTION
**Pull request by rtfb**
I went through README and the docs and picked out some remaining inaccuracies.

### CODE REVIEW

File: html.go [hunk 1]
```diff
@@ -482,8 +482,8 @@ func (r *HTMLRenderer) outHRTag(w io.Writer) {
 }
 
 // RenderNode is a default renderer of a single node of a syntax tree. For
-// block nodes it will be called twice: first time with entering=true, second
-// time with entering=false, so that it could know when it's working on an open
+// block nodes it will be called twice: first time with entering=True, second
+// time with entering=False, so that it could know when it's working on an open
```
Comment (shurcooL) [inline html.go]: Why this change? Isn't lower case correct? https://godoc.org/builtin#true

File: README.md [hunk 2]
```diff
@@ -52,22 +52,27 @@ Potential drawbacks:
...
 
 Usage
 -----
 
 For the most sensible markdown processing, it is as simple as getting your input
 into a byte slice and calling:
 
+``` go
     output := blackfriday.Run(input)
+```
```
Comment (shurcooL) [inline README.md]: When you go from indented code block to fenced code block, you should get rid of the indentation, since it'll now be a part of the code block. This should become:

    ```go
    output := blackfriday.Run(input)
    ```

Also, it's more common not to include a space after the three backticks.

File: html.go (hunk 1 above)
Comment (rtfb) [inline html.go]: Wow, that's a brain fart. Obviously too much Python lately :-). I'll fix it, thanks!

File: README.md (hunk 2 above)
Comment (rtfb) [inline README.md]: Will fix, thanks.
